server/sonar-server/src/test/java/org/sonar/server/ws/WsActionTester.java

```diff
@@ -44,4 +45,14 @@ public TestRequest newRequest() {
     request.setAction(action);
     return request;
   }
+
+  public URL getTestFile(Class clazz, String fileName) {
```

Comment:
1. This method as nothing to do here (it's not specific to WS test)
2. No need for a helper for that, google is your friend ! (see my comment on UserActionTest.search_for_users_with_one_permission)